### DISCUSSION TITLE
require("neotest").run is nil

### DISCUSSION THREAD

**Discussion by bendavis78** [nvim-neotest/neotest#399] [Q&A]
I have `nvim-neotest/neotest` and `nvim-neotest/neotest-pthon` installed via Lazy (confirmed when viewing `:Lazy`.  When I run `require("neotest").run.run()`, apparently the first `run` is nil.  
```
:lua require('neotest').run.run()
E5108: Error executing lua [string ":lua"]:1: attempt to index field 'run' (a nil value)
stack traceback:
        [string ":lua"]:1: in main chunk
```

My configs look like this:

plugins.lua
```lua
...
	-- NeoTest
	{
		"nvim-neotest/neotest",
		dependencies = {
			"nvim-neotest/nvim-nio",
			"nvim-lua/plenary.nvim",
			"antoinemadec/FixCursorHold.nvim",
			"nvim-treesitter/nvim-treesitter",
		},
		config = require("configs.neotest"),
	},
	{ "nvim-neotest/neotest-python" },
...
```

configs/neotest.lua:
```lua
return function()
  require("neotest").setup({
    adapters = {
      require("neotest-python")({
        dap = { justMyCode = false },
      }),
    }
  })
end
```

**Comment by codymikol** [comment]
If you want to run a neotest spec, you run the command `:Neotest run` I don't think there is a run function on neotest

**Comment by YeritsyanTigran** [comment]
Is there any update here? Currently trying to use the lazy config of neovim from their website and I have the same error. Readme says that there should be neotest.run
